css/at-rules/media.json

```diff
@@ -885,6 +885,114 @@
             }
           }
         },
+        "overflow-block": {
+          "__compat": {
+            "description": "<code>overflow-block</code> media feature",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/overflow-block",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
```

Comment:
I wonder if it is fair to assume that no browser has already implemented these (very recent) feature. Then we could just initialize everything with `false`.